xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/pvr/Recordings.h

```diff
@@ -56,6 +56,7 @@ class PVRRecording : public DynamicCStructHdl<PVRRecording, PVR_RECORDING>
     m_cStructure->iChannelUid = PVR_RECORDING_VALUE_NOT_AVAILABLE;
     m_cStructure->channelType = PVR_RECORDING_CHANNEL_TYPE_UNKNOWN;
     m_cStructure->sizeInBytes = PVR_RECORDING_VALUE_NOT_AVAILABLE;
+    m_cStructure->iParentalRating = 0;
```

Comment:
This can be omitted, as the struct is always initialized with zeros.

Reply:
Done.